.github/ISSUE_TEMPLATE/new_component.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Add a New Bento UI Component
+about: Add a New Seearch Results Component to UI
+title: 'Add [Name] Component to UI''
+labels: ''
+assignees: ''
+
+---
+
+## Description
+What the component displays
+
+### All Search API Information
+What call you need to make to populate the component with results.
+
+### Supporting Service Documentation
+Other Background Information.
+
+### Labeling
+What heading should be used for the component. 
+
+### Acceptance Criteria
+
+- [ ] Displays a heading
+- [ ] Display an icon next to the heading relevant to the component type (i.e a book for the catalog)
+- [ ] Displays up to three search results. 
+- [ ] Provides a link to see more results in the source user interface.
+- [ ] Displays the number of total results in the source interface. 
+- [ ] Passes WCAG 2.1 Audit
```

Comment:
Could we clarify the scope of the accessibility checking?  Should we go through the whole audit procedure, or just do the voiceover + browser extension testing that we include in the other issue templates?

Reply:
Let's do what we have in the other templates. Thanks for pointing that out.